### ISSUE TITLE
Font not found after adding private metanorma repo

### ISSUE THREAD

**Issue by CAMOBAP** [fontist/fontist#272]
Recently we faced with the issue [[fontist] 'Tahoma' font is not supported. ...](https://github.com/metanorma/mn-samples-bsi/runs/4248402206?check_suite_focus=true#step:9:409) in private repo mn-samples-bsi

```
  fontist repo list
  fontist repo setup metanorma ***github.com/metanorma/fontist-formulas-private
  fontist repo update metanorma
  fontist repo list
  fontist install Tahoma
  shell: sh -e {0}
Fontist repo 'metanorma' from '***github.com/metanorma/fontist-formulas-private' has been successfully set up.
Fontist repo 'metanorma' has been successfully updated.
metanorma
Font "Tahoma" not found locally.
Font 'Tahoma' not found locally nor available in the Fontist formula repository.
Perhaps it is available at the latest Fontist formula repository.
You can update the formula repository using the command `fontist update` and try again.
```

@alexeymorozov maybe you have an idea what is wrong

**Comment by alexeymorozov** [comment]
Released v1.11.7 with the fix.

The bug was caused by compatibility with the `fontist repo setup` command:
1. `fontist repo setup` created a formulas dir but without a main git repo
2. `fontist update` checked that the dir is already created and tried to update it
3. and failed because this dir had no `.git` dir

**Comment by ronaldtse** [comment]
Thank you @alexeymorozov !
